README.md

```diff
@@ -19,3 +19,5 @@ Provides:
 - `yq` v4.35.1
 - `tini` v0.19.0
 - `rclone`
+- `ibm-watson-machine-learning`
```

Comment:
it doesn't install ibm-watson-machine-learning, so it should be removed from the readme

Reply:
Hi @whitfiea 
It is fixed now 

Thank you